lib/ExpandablePicker/ExpandablePicker.less

```diff
@@ -38,7 +38,7 @@
 
 /* Current Value */
 .moon-expandable-picker-current-value {
-	.moon-body-text;
+	.moon-item-body-text;
```

Comment:
Thanks! Will we need to add a rule like this for `moon-item-body-text` instead? https://github.com/enyojs/moonstone/blob/2.6.0-dev/css/moonstone-text.less#L179

The line-height is now `39px` in both non-latin and other locales.